src/FunctionLoader.cs

```diff
@@ -57,6 +58,10 @@ internal static void SetupWellKnownPaths(FunctionLoadRequest request)
             var appLevelModulesPath = Path.Join(FunctionAppRootPath, "Modules");
             var workerLevelModulesPath = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "Modules");
             FunctionModulePath = $"{appLevelModulesPath}{Path.PathSeparator}{workerLevelModulesPath}";
+            if (Platform.IsWindows)
```

Comment:
Rather than using `Platform.IsWindows` can we check if an environment variable is present that only exists in App Service Sandbox?

Reply:
Do you see an obvious benefit of doing this over checking for platform? I'm don't know the details yet.

Reply:
If the user is running their function app locally, this path will be added to the module path if they're on Windows.

That's not ideal because this path is specific to the App Service sandbox.

Reply:
Do you've an example usage of this in PS language worker, that differentiate if it's App service windows sandbox of Linux app service environment?

Reply:
I don't, unfortunately.

@pragnagopa @fabiocav do one of you have an example environment variable that would _only_ apply to the function app sandbox?

Reply:
`WEBSITE_INSTANCE_ID` is what we use to check if we're running in App Service.

Reply:
@fabiocav WEBSITE_INSTANCE_ID environment variable is sufficient to check if this is Window App Service or Linux app service environment?

Reply:
@SatishRanjan you can see the logic we use for all of those checks here:
https://github.com/fabiocav/azure-functions-host/blob/2fc6590888f6bcc38402eb80645701eb02a3f5ef/src/WebJobs.Script/Environment/EnvironmentExtensions.cs#L15-L28